src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkFromMyCollection.tests.tsx

```diff
@@ -83,6 +83,8 @@ const mockedFetchedArtwork: NonNullable<FetchArtworkInformationResult> = {
     state: "Berlin",
     country: "Germany",
     postalCode: "12345",
+    address: "Berlin",
+    address2: "",
```

Comment:
Might be good to test if the form initializes `address2` as well.

```suggestion
    address2: "Something",
```

Reply:
done

Reply:
But not yet committed 🤔

Reply:
pushing it now